File: java/UberAdvanced.java
import java.util.ArrayList;
import java.util.Map;

class UberAdvanced extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberAdvanced (Integer id, String license, Account driver, Integer passengers, Map<String, Map<String, Integer>> typeCarAccepted, ArrayList<String> seatsMaterial) {
        super(id, license, driver, passengers);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: php/uberAdvanced.php
<?php
    require_once("./car.php");
    class UberAdvanced extends Car {
        public $typeCarAccepted;
        public $seatsMaterial;

        public function __construct ($id, $license, $driver, $passengers, $typeCarAccepted, $seatsMaterial) {
            parent::__construct($id, $license, $driver, $passengers);

            $this->typeCarAccepted = $typeCarAccepted;
            $this->seatsMaterial = $seatsMaterial;
        }
    }
?>